docker/compose/validator-testnet/validator_node_template.yaml

```diff
@@ -0,0 +1,32 @@
+consensus:
+  # The following slow consensus down when there are no active transactions to process
+  round_initial_timeout_ms: 20000
+  mempool_poll_count: 333
+
+# The rest of this config is copy paste of config/src/config/test_data/validator.yaml
+validator_network:
+    listen_address: "/ip4/0.0.0.0/tcp/6180"
+    identity:
+        type: "from_storage"
```

Comment:
Your spacing is fairly inconsistent 2 spaces in one section and 4 in another. I was under the impression that 2 spaces is typically using in yaml files but I could be wrong.